napalm/ios.py

```diff
@@ -822,3 +822,15 @@ def get_environment(self):
         environment.setdefault('temperature', {})
         environment['temperature']['invalid'] = {'is_alert': False, 'is_critical': False, 'temperature': -1.0,}
         return environment
+
+    def cli(self, command=''):
```

Comment:
Could you revisit this PR and fix it so it complies with #156? Some late changes we did were:

1. command was renamed to commands and the default argument is None
1. If commands is not a list it should raise a TypeError exception
1. Because commands is not a list of commands all of them should be executed.